modules/ROOT/pages/access-recovery.adoc

```diff
@@ -0,0 +1,10 @@
+= Access Recovery
+
+If you've lost the private key of an SSH key pair used to log into a Fedora system, and do not have any password logins set up to use at the console, you can gain access back to the machine by booting into single user mode with the `single` kernel command-line argument:
```

Comment:
when trying this I got:

```
Cannot open access to the console, the root account is locked.
See sulogin(8) man page for more details.

Press Enter to continue.
```

Reply:
Thanks for testing this.  Yes it's of course because I didn't carry over the SULOGIN_FORCE bits we added a while ago.  For now I did https://gitlab.com/bootc-org/fedora-bootc/docs/-/merge_requests/4